docs/scos/dev/upgradability-services/new-documentation/core-method-used-on-project-level.md

```diff
@@ -0,0 +1,83 @@
+---
+title: A core method is used on the project level
+description: Reference information for evaluator and upgrader tools.
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+## A core method is used on the project level
+
+Modules have public and private APIs. The public API includes the entities like facade, plugin stack, dependency list, etc. This check covers private API entities, including Business model, Factory, Dependency provider, Repository, and Entity manager.
```

Comment:
The best would be to have a link to a spryker doc where described private/public API

Reply:
Like to get more information - see here